asv2/privacy.md

```diff
@@ -0,0 +1,310 @@
+```
+</details>
+
+## Using the SDK
+
+The [`@celo/identity`](https://www.npmjs.com/package/@celo/identity) package is an SDK that provides useful helper functions to handle querying ODIS and deriving the obfuscated identifier.
+
+> ❗️ Make sure you are using version `3.0.1` or later
+
+Import the `OdisUtils` module into your project for all relevant functions
+```typescript
+import { OdisUtils } from "@celo/identity";
+```
+
+-------------
+The most important function is `getObfuscatedIdentifier`, which completes the entire process of obfuscated identifier derivation, starting from the plaintext identifier.
+
+#### Parameters:
+
+|Parameter | Type | Description |
+|------ | ------ | ----------------------- |
+`plaintextIdentifier` | string | plaintext identifier to be obfuscated
+`identifierPrefix`| IdentifierPrefix | you can use a predefined type on the `IdentifierPrefix` enum or you can cast an arbitrary string
+`account` | string | account making the ODIS query that quota will be deducted from, see [Rate Limit](#rate-limit)
+`signer` | AuthSigner | object describing the authentication method and providing authentication key, see [Authentication](#authentication)
+`context` | ServiceContext | object providing the ODIS context, see [Service Context](#service-context)
+`blindingFactor` (optional) | string | secret seed used for blinding/unblinding the identifier, by default a one-time random seed is used in the blinding client
+`clientVersion` (optional) | string | 
+`blsBlindingClient` (optional) | BlsBlindingClient | the default blinding client used only works server-side, see [Runtime Environments](#runtime-environments) for alternatives
+`sessionID` (optional) | string | 
+`keyVersion` (optional) | number | 
+`endpoint` (optional) | | 
+
+#### Returns: 
+`Promise‹IdentifierHashDetails›`
+
+```typescript
+interface IdentifierHashDetails {
+  plaintextIdentifier: string
+  obfuscatedIdentifier: string
+  pepper: string
+  unblindedSignature: string
+}
+```
+
+#### Example:
+
+```typescript
+import { OdisUtils } from "@celo/identity";
+
+await OdisUtils.Identifier.getObfuscatedIdentifier(
+  '+12345678910',
+  OdisUtils.Identifier.IdentifierPrefix.PHONE_NUMBER,
+  issuerAddress,
+  authSigner,
+  serviceContext
+)
+// {
+//   plaintextIdentifier: '+12345678910',
+//   obfuscatedIdentifier: 'c1fbb1429e94f4...3f1a',
+//   pepper: '123abc',
+//   unblindedSignature: '9as8duf98as...df80u'
+// }
+```
+*Source code for `getObfuscatedIdentifier` and other relevant functions [here](https://github.com/celo-org/celo-monorepo/blob/master/packages/sdk/identity/src/odis/identifier.ts)*
+
+-------------
+
+### Rate Limit
+
+ODIS implements rate limiting on queries to prevent brute force attackers. Every account interacting with ODIS has a quota for the number of queries it can make.
+
+You can check how much quota is left on your account:
+
+```typescript
+const { remainingQuota } = await OdisUtils.Quota.getPnpQuotaStatus(
```

Comment:
this might be a good place to also include (or link to) instructions on how to top up quota

Reply:
I think querying remaining quota and topping up should be grouped together in the docs since most wallets will set up a separate process to manage odis quota